.github/workflows/linters.yml

```diff
@@ -0,0 +1,47 @@
+name: Linters
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  NODE_VERSION: lts/*
+
+permissions:
+  contents: read
+
+jobs:
+  lint-js-and-md:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - name: Environment Information
+        run: npx envinfo
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "dir=$(corepack yarn config get cacheFolder)" >> $GITHUB_OUTPUT
+
+      - uses: actions/cache@v3
```

Comment:
I dont think this is needed as caching is built into setup action
https://github.com/actions/setup-node#caching-global-packages-data

Reply:
I copied it from Uppy repo. IIRC we had to add this step because the cache only worked for Yarn v1, are we sure it’s going to work on more modern versions? I’d rather keep the step even if it turns out to be redundant than wasting time on every run because caching doesn’t work.